tests/flow/test_index_create.py

```diff
@@ -173,3 +183,6 @@ def test04_index_creation_pattern_syntax(self):
         result = redis_graph.query("CREATE INDEX FOR ()-[r:follow]-() ON (r.prop1, r.prop2)")
         self.env.assertEquals(result.indices_created, 2)
 
+    def test05_index_delete(self):
+        pool = Pool(nodes=10)
+        pool.map(issue_query, range(1, 100))
```

Comment:
I don't think `issue_query` is a suitable name for what we're doing here, please consider renaming to `create_drop_index` also because this test function is the only one calling it consider declaring this utility function within `test05_index_delete`